docs/manual-EN/2.query-language/3.language-structure/user-defined-variables.md

```diff
@@ -0,0 +1,16 @@
+# User-Defined Variables
+
+**Nebula Graph** supports the concept of user-defined variables, which allows passing of a value from one statement to another. A user-defined variable in written as `$ var_name`, `var_name` is the name of variable and can consist of alphanumeric characters, any other characters are not recommended currently.
+
+User-defined variables can only be used in one execution (compound statements separated by semicolon (;) or pipe (|) and be submitted to server to execute together).
```

Comment:
User-defined variables can only be used in one execution (compound statements separated by semicolon (;) or pipe (|) and is submitted to the server to execute together).